tests/examples/pacer/nef/s3/ared_1.json

```diff
@@ -0,0 +1,50 @@
+{
+  "case_name": "Fuller v. Singleton Farms LLC",
+  "court_id": "ared",
+  "date_filed": "2021-06-28",
+  "docket_entries": [
+    {
+      "date_filed": "2021-06-28",
+      "description": "RESPONSE re [11] Statement of Facts (Local Rule 56.1)",
+      "document_number": "13",
+      "pacer_doc_id": "02705225168",
```

Comment:
We should get the sequence number from the `de_seq_num=3D40` part of the message, and we'll need the `magic_num` too so we can download it by reconstructing the link ourselves, or else we need the whole link somewhere in our JSON.

Reply:
Maybe it makes sense for the parser to provide both deconstructed and the entire link. Consumers make the choice?

Reply:
Yes, added utils for those, also added the full document URL which ends up being quite useful in its own right.